java-frontend/src/main/java/org/sonar/plugins/java/api/tree/MethodTree.java

```diff
@@ -90,4 +88,10 @@
 
   Symbol.MethodSymbol symbol();
 
+  /**
+   * Check if a methodTree is overriding any other method. The corresponding overridden symbol can be retrieved through the {@link #symbol()}.
+   *
+   * @return true if overriding, null if it cannot be decided (method symbol not resolved or lack of byte code for super types), false if not overriding.
+   */
+  Boolean isOverriding();
```

Comment:
isn't this lacking an annotation  ?

Reply:
Good catch! I forgot the `@CheckForNull` annotation!